src/modules/pmc/src/lib/PmcBase.h

```diff
@@ -45,17 +45,18 @@ class PmcBase
 public:
     struct DesiredState
     {
-        std::vector<std::string> packages;
-        std::map<std::string, std::string> sources;
+        std::vector<std::string> Packages;
+        std::map<std::string, std::string> Sources;
+        std::map<std::string, std::string> GpgKeys;
     };
 
     struct State
     {
-        ExecutionState executionState;
-        std::string packagesFingerprint;
-        std::vector<std::string> packages;
-        std::string sourcesFingerprint;
-        std::vector<std::string> sourcesFilenames;
+        ::ExecutionState ExecutionState;
```

Comment:
It's a weird C++ restriction that doesn't allow to use ExecutionState as field name because it collides with the class name. With having a :: in front, I tell the compiler that ExecutionState is the class from the anonymous namespace and then it is able to differentiate the field name from the class name.